### ISSUE TITLE
Thermoelectric Generator accepts Lava Bucket only when tank is empty

### ISSUE THREAD

**Issue by Krazy-J** [ICY105/Mechanization#142]
The Thermoelectric Generator only accepts a Lava Bucket when the second tank is empty. The execute command on line 13 of the function `mechanization:machines/machines/thermoelectric_generator/tank_2_input` is checking the wrong data.
```mcfunction
execute ... if data entity @s Item.tag.tank.tag.ctc.traits.liquid{id:"lava"} run ...
```
"tank" should be "tank_2".

**Comment by Krazy-J** [comment]
I found a few more issues with the Thermoelectric Generator's tanks. tank_1_input.json `mechanization:machines/machines/thermoelectric_generator/tank_1_input` also references "tank" when it should be "tank_1" on lines 13, 15, and 17.

Additionally, `milk_bucket` is not in the function. Right now, the only way to put Milk in the Thermoelectric Generator is to put a Milk Bucket in a different machine/tank, take the milk out with a vial, and then put that in the generator.

**Comment by ICY105** [comment]
Fixed in 3.0.5

**Comment by misterx** [comment]
The same for water now. Generator takes bucket of water when first tank is empty
